eth/contracts/facets/DFLobbyFacet.sol

```diff
@@ -20,7 +20,7 @@ contract DFLobbyFacet is WithStorage {
         address diamondCutAddress = IDiamondLoupe(diamondAddress).facetAddress(
             IDiamondCut.diamondCut.selector
         );
-        Diamond lobby = new Diamond(diamondAddress, diamondCutAddress);
+        Diamond lobby = new Diamond();
```

Comment:
How is the address set if you don't pass it in here? I think you need to add your own constructor in the inherited diamond, right?

Reply:
Oh, I vaguely remember that the Diamond provided by SolidState has the facet cut stuff embedded into the contract. Maybe that's why you don't need those 2 variables?

Reply:
Afaict, the new Diamond implementation sets owner to be `msg.sender` and includes the DiamondCut function as part of the deploy: https://github.com/solidstate-network/solidstate-solidity/blob/690f3058c614961227b27f342b43996336dc239d/contracts/proxy/diamond/SolidStateDiamond.sol#L35

So I could add a wrapper constructor, but I don't think (?) it's necessary